src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -159,5 +166,8 @@ export default compose(
         selectedTab: {
             key: `${ONYXKEYS.COLLECTION.SELECTED_TAB}${CONST.TAB.RECEIPT_TAB_ID}`,
         },
+        policy: {
+            key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${lodashGet(report, 'policyID')}`,
```

Comment:
I think just to be safe we should add 0 here, thats because that in case no policyID is found then we are not loading the entire collection
```suggestion
            key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${lodashGet(report, 'policyID', '0')}`,
```

Reply:
Ah ok, I thought the default would be an empty string